components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -4,225 +4,9 @@
-            title = (
-                <FormattedMessage
-                    id='admin.billing.subscription.privateCloudCard.cloudProfessional.title'
-                    defaultMessage='Upgrade to Cloud Enterprise'
-                />
-            );
-            description = (
-                <FormattedMessage
-                    id='admin.billing.subscription.privateCloudCard.cloudProfessional.description'
-                    defaultMessage='Advanced security and compliance features with premium support. See {pricingLink} for more details.'
-                    values={{pricingLink}}
-                />
-            );
-            break;
-        }
-    }
-
-    return (
-        <div className='PrivateCloudCard'>
-            <div className='PrivateCloudCard__text'>
-                <div className='PrivateCloudCard__text-title'>
-                    {title}
-                </div>
-                <div className='PrivateCloudCard__text-description'>
-                    {description}
-                </div>
-                {(isFreeTrial || subscriptionPlan === CloudProducts.ENTERPRISE || isCloudLegacyPlan) &&
-                    <a
-                        href={isFreeTrial ? trialQuestionsLink : contactSalesLink}
-                        rel='noopener noreferrer'
-                        target='_blank'
-                        className='PrivateCloudCard__actionButton'
-                        onClick={() => trackEvent('cloud_admin', 'click_contact_sales')}
-                    >
-                        <FormattedMessage
-                            id='admin.billing.subscription.privateCloudCard.contactSales'
-                            defaultMessage='Contact Sales'
-                        />
-
-                    </a>
-                }
-                {(!isFreeTrial && productsLength > 1 && subscriptionPlan !== CloudProducts.ENTERPRISE && subscriptionPlan !== CloudProducts.LEGACY) &&
-                    <button
-                        type='button'
-                        onClick={onUpgradeMattermostCloud}
-                        className='PrivateCloudCard__actionButton'
-                    >
-                        <FormattedMessage
-                            id='admin.billing.subscription.privateCloudCard.upgradeNow'
-                            defaultMessage='Upgrade Now'
-                        />
-                    </button>
-                }
-            </div>
-            <div className='PrivateCloudCard__image'>
-                {isFreeTrial ?
-                    <CloudTrialSvg
-                        width={170}
-                        height={123}
-                    /> :
-                    <PrivateCloudSvg
-                        width={170}
-                        height={123}
-                    />
-                }
-            </div>
-        </div>
-    );
-};
-
-export const cancelSubscription = (cancelAccountLink: any, isFreeTrial: boolean, isPaidTier: boolean) => {
```

Comment:
I moved this function/component to `components/admin_console/billing/billing_subscriptions/cancel_subscription.tsx` because I found it distracting while I was trying to dig in and understand how code on this page worked. No other changes were made to this component.